pkgs/desktops/gnome-2/platform/libgnomeprint/default.nix

```
@@ -9,10 +9,16 @@ stdenv.mkDerivation rec {
     sha256 = "1034ec8651051f84d2424e7a1da61c530422cc20ce5b2d9e107e1e46778d9691";
   };
 
-  patches = [ ./bug653388.patch ];
+  patches = [
+    ./bug653388.patch
+    (fetchpatch {
```

Also can we add a comment:

```suggestion

    # Fix compatibility with bison 3
    (fetchpatch {
```